docs/user/api/_index.md

```diff
@@ -176,14 +176,14 @@ Response on success:
 }
 ```
 
-### List current scrape targets
+### List current scrape targets of metrics subsystem
 
 ```
-GET /agent/api/v1/targets
+GET /agent/api/v1/metrics/targets
```

Comment:
We can also change another reference to the 'old' endpoint on this file, on line 350.